webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -78,8 +101,88 @@ HostInstalledProfiles.propTypes = {
   installedProfiles: PropTypes.arrayOf(PropTypes.string).isRequired,
 };
 
+const performModuleStreamAction = (hostName, action, moduleSpec, dispatch) => {
+  dispatch(moduleStreamAction({ hostname: hostName, action, moduleSpec }));
+};
+
+const ModuleActionConfirmationModal = ({
+  hostName, action, moduleSpec, actionModalOpen, setActionModalOpen,
+}) => {
+  const dispatch = useDispatch();
+  let title;
+  let body;
+  let confirmText;
+  switch (action) {
+  case 'disable':
+    confirmText = __('Disable');
+    title = __('Disable module stream');
+    body = __(`Selected module ${moduleSpec} will become unavailable. 
+      The module RPMs will become unavailable in the package set.`);
+    break;
+  case 'reset':
+    confirmText = __('Reset');
+    title = __('Reset module stream');
+    body = __(`Selected module ${moduleSpec} will be no longer enabled or disabled. 
```

Comment:
We're not supposed to use string interpolation with translation..

I think these will need to use `<FormattedMessage>`. (shouldn't be too bad ;)